server/index.js

```diff
@@ -20,7 +22,17 @@ app.get("/", (req, res) => {
 });
 
 app.use("/", earth911Router);
+app.use("/auth", authRoutes);
 
-app.listen(PORT, () => {
-  console.log(`EcoHabit is running, server listening to ${PORT}`);
-});
+mongoose.set("strictQuery", true);
+mongoose
+  .connect(process.env.MONGO_URL, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  })
+  .then(() => {
+    app.listen(PORT, () =>
+      console.log(`EcoHabit is running, server listening to ${PORT}`)
+    );
+  })
+  .catch((error) => console.log(`${error} did not connect`));
```

Comment:
a database connection is already maintained in `95-db-and-models-setup` branch and is currently in a PR.